### DISCUSSION TITLE
High usage on list after enable tier.

### DISCUSSION THREAD

**Discussion by OrvilleQ** [minio/minio#17731] [Q&A] [answered]
After enable minio tier on my backblaze storage, I got 480K S3 List Object request in just a week.

![image](https://github.com/minio/minio/assets/21377465/aa739156-ad08-4f42-9a73-88671dc200ad)

Is that normal? If true, is it possible to redurce that amount of request?

**Comment by harshavardhana** [comment]
Which version of MinIO is this?

**Reply by OrvilleQ** [reply, marked as answer]
I figured it out. It was a misconfigured csi backup on the minio pod that was causing the issue, not minio.

I'll close this discussion, thanks for your reply.
